lua/kubectl/actions/commands.lua

```diff
@@ -146,14 +147,16 @@ function M.load_config(file_name)
   if ok then
     return decoded
   end
-  return nil
+  return { session = { view = "pods", namespace = "All" }, filter_history = {} }
```

Comment:
This adds a connection to the content/model of the config file, I'd rather have the state/session_manager own that.
The idea is to follow the SOLID principle (The S in solid, separation of responsibility), so we should return nil here and let the state/session_manager decide what that means

Reply:
ok returning `nil` works without changing anything